pkg/apis/core/v1beta1/types_seed.go

```diff
@@ -224,6 +226,9 @@ type SeedProvider struct {
 	ProviderConfig *runtime.RawExtension `json:"providerConfig,omitempty" protobuf:"bytes,2,opt,name=providerConfig"`
 	// Region is a name of a region.
 	Region string `json:"region" protobuf:"bytes,3,opt,name=region"`
+	// Zones is the list of availability zones the seed cluster is deployed to.
+	// +optional
+	Zones []string `json:"zones,omitempty" protobuf:"bytes,4,rep,name=zones"`
```

Comment:
Should this optional?
All seeds will have at least 1 zone.

Reply:
To be backwards-compatible, newly introduced fields in the API always have to be optional. Also, it's probably reasonable to make such assumption given the large variety of infrastructure providers (I can imagine that regions w/o zones exist).